app.py

```diff
@@ -0,0 +1,139 @@
+from flask import Flask, request, abort
+from git import Repo
+import ansible_runner
+import os
+import requests
+import json
+import configparser
+
+from datetime import datetime
+
+app = Flask(__name__)
+
+def appendTime(message):
+    output = "[" + datetime.utcnow().isoformat() + "]" + " " + message
+    return (output)
+
+def updateRepo(repoName, repoURL): #Update the local git repo
+
+    if os.path.isdir(repoName): #If the clone of the repo exists pull updates, if not clone it 
+        repo = Repo(repoName)
+        o = repo.remotes.origin
+        o.pull()
+        print(appendTime("Repo Pulled"))
+    else:
+        Repo.clone_from(repoURL, repoName)
+        print(appendTime("Repo Cloned"))
+
+def fileChanges(commits): #Returns bool for if vlan file changes and list of individual switch changes
+    
+    vlanChange = False #Vlan Flag
+    switchChanges = [] #List of modified switches
+
+    for commit in commits: #Look at each commit in push to main branch
+        
+        for file in commit['modified']: #Look at the files modified in each commit
```

Comment:
What if a new switch has been added to the configuration? You probably want to look at both `commit['modified']` and `commit['added']`.